packages/common/base/package.json

```diff
@@ -15,8 +15,8 @@
     "types": "./dist/index.d.ts",
     "files": ["dist", "src", "LICENSE"],
     "scripts": {
-        "build": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --minify --dts --sourcemap",
-        "dev": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --dts --sourcemap --watch",
+        "build": "tsup --minify",
+        "dev": "tsup --watch",
```

Comment:
should we keep generating sourcemaps in `dev` for ease of development?

Reply:
i can add them to `dev`, but we really need a proper solution - where during the build for release (during the ci/cd) we dont generate sourcemaps, but locally we do

i've already made this issue here to stop including sourcemaps in the prod deploy
https://linear.app/crossmint/issue/INF-625/sdk-repo-dont-publish-maps-to-final-dist

Reply:
Agreed, but we should have them in dev, they're super useful for debugging in dev tools

Reply:
check [new commit](https://github.com/Crossmint/crossmint-sdk/pull/788/commits/7691a2b8a2c4ccf568f10cbd70423baecb15733f)